gvsbuild/projects.py

```diff
@@ -51,11 +50,13 @@ def build(self):
 
         self.push_location(r".\win32")
         self.exec_vs(
-            r'nmake /nologo /f adwaita-msvc.mak CFG=%(configuration)s PYTHON="%(python_dir)s\python.exe" PREFIX="%(gtk_dir)s"',
```

Comment:
can we increase the line size?

Reply:
While it is possible to increase the line length, I think we should just stick to the Black defaults. Here is the [full description](https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#line-length) of why line length is set to 88, but the tldr of it is that it is trying to get close to PEP8 of 79, with some extra wiggle room to reduce file length. The positive is that Black does a great job of wrapping lines when they are too long.

Reply:
while I normally I agree about it, this is wrapping even tarball links which is just bad... I wonder if we can specify that strings should not be wrapped

Reply:
@nacho I looked at this closer, and you were right - thanks for catching this! I was able to undo breaking up the raw strings while still allowing black and flake8 to be happy. 👍